transport/http/codes.go

```diff
@@ -60,6 +60,7 @@ var (
 			yarpcerrors.CodeAborted,
 			yarpcerrors.CodeAlreadyExists,
 		},
+		422: {yarpcerrors.CodeInvalidArgument},
```

Comment:
I am afraid it wouldn't solve the problem of the reverse mapping. thrift(edge)->http->grpc requires us to return yarpcError. Then it is being mapped to http status. We seem to need the opposite conversion `_codeToStatusCode`. Look at this studio [request](https://studio.uberinternal.com/requests/ee952446-e8fa-431d-9c63-cd3bb171b760). We call grpc service via http. Here is the [implementation](https://sourcegraph.uberinternal.com/-/editor?remote_url=gitolite%40code.uber.internal%3Ago-code&branch=main&file=src%2Fcode.uber.internal%2Fdirect%2Fpresentation%2Fhandler%2Fdaas_delivery%2Fdelivery.go&start_row=95&start_col=23&end_row=95&end_col=23&editor=JetBrains&version=v2.1.2). We just return an yarpc error and get 400 mapped from it